CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,116 @@
+# CFPB Open Source Code of Conduct
+
+## Introduction
+
+The [Consumer Financial Protection Bureau](https://www.consumerfinance.gov) (CFPB) is committed to 
+building a safe, welcoming, harassment-free culture for everyone. We do not merely want an 
+environment that is free from hostility; we want one that is actively welcoming and inclusive. We 
+want our team, our workplace culture, and our open source community to reflect and celebrate the 
+diversity of the people we serve.
+
+This Code of Conduct summarizes federal anti-harassment law and CFPB policy. 
+Scope
+
+We expect everyone on the CFPB team, and those contributing to our open source community, to exhibit 
+these behaviors and abide by applicable federal laws and CFPB policies. In addition, we expect 
+everyone within CFPB spaces to exhibit these behaviors and refrain from behavior prohibited by 
+anti-harassment laws and federal policies on harassment. These spaces include:
+
+- CFPB’s physical offices,
+- CFPB events and meetings,
+- All of CFPB’s online forums and virtual collaboration tools, including code repositories
+
+
+## What we strive for
+
+At CFPB, we strive to create a welcoming and inclusive culture that empowers people to best protect 
+the financial interests of all consumers. That kind of atmosphere requires an open exchange of ideas 
+balanced by thoughtful guidelines. Examples of behavior that contributes to a positive environment 
+for our open source community include:
+
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+- Focusing on what is best not just for us as individuals, but for the overall community and public
+
+## Unacceptable behavior
+
+To help understand the kinds of behaviors that are illegal or run counter to the culture we seek to 
+foster, we've listed below actions that violate federal law and CFPB policy. We've also included 
```

Comment:
I'm a little uncomfortable with the inclusion of "illegal" and "violate federal law" in this sentence, without making it clear below which of these things might be actual violations of law. The lists below only refer to our policy.

Reply:
This is in line with the wording of the [18F/TTS code of conduct](https://18f.gsa.gov/code-of-conduct/), but noted for a future iteration.